### DISCUSSION TITLE
Lychee 6.1.1 after uploading images aren't shown

### DISCUSSION THREAD

**Discussion by prbt2016** [LycheeOrg/Lychee#2748] [Bugs] [answered]
### Lychee version

6.1.1

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

Hello @ildyria ,

I was in the process of manual installation of Lychee 6.1.1 downloaded from https://github.com/LycheeOrg/Lychee/releases/download/v6.1.1/Lychee.zip on CentOS Linux release 7.9.2009 (Core) with PHP 8.2, MYSQL 8.0, Apache 2.4.

However, after uploading image under any of the 'Smart albums' no image is displayed , only placeholder with image name is shown , as shown in screenshot : 

![lychee](https://github.com/user-attachments/assets/ab48b039-5025-4980-a6a4-af19261d79be)

On clicking the blank placeholder image, the actual image is shown , but not when visiting the album: 

![lychee1](https://github.com/user-attachments/assets/e340c33f-1275-40b0-b107-9dd88d585989)

Following is the output of error.log inside /storage

```
[DATE] production.ERROR: Imagick::writeImageFile:93 delegate failed `'cwebp' -quiet %Q '%i' -o '%o'' @ error/delegate.c/InvokeDelegate/1928
App\Image\Handlers\ImagickHandler::save:100 Failed to save image; caused by
```

What could be the cause of issue , could you please replicate this at your end ?.

Regards.

### Steps to reproduce the issue

1. Install Lychee using https://github.com/LycheeOrg/Lychee/releases/download/v6.1.1/Lychee.zip
2. Go to any 'Smart albums' -> Click on upload
3. No image is shown

### Diagnostics [REQUIRED]

System Information
------------------
Lychee Version (release):                  6.1.1
DB Version:                                6.1.1

composer install:                          --no-dev
APP_ENV:                                   production
APP_DEBUG:                                 false
APP_URL:                                   set
APP_DIR:                                   set
LOG_VIEWER_ENABLED:                        true
VUEJS_ENABLED:                             true
PHOTO_PIPES:                               true

System:                                    Linux
PHP Version:                               8.2.8
PHP User agent:                            Lychee/4 (https://lycheeorg.github.io/)
Timezone:                                  UTC
Max uploaded file size:                    2M
Max post size:                             8M
Chunk size:                                2.00 MB
Max execution time:                        0
MySQL Version:                             8.0.40

exec() Available:                          yes
Imagick Available:                         1
Imagick Enabled:                           1
Imagick Version:                           1690
GD Version:                                bundled (2.1.0 compatible)
Number of foreign key:                     3198 found.

Config Information
------------------
version:                                   060101
check_for_updates:                         0
sorting_photos_col:                        taken_at
sorting_photos_order:                      ASC
sorting_albums_col:                        max_taken_at
sorting_albums_order:                      ASC
imagick:                                   1
skip_duplicates:                           0
small_max_width:                           0
small_max_height:                          360
medium_max_width:                          1920
medium_max_height:                         1080
lang:                                      en
image_overlay_type:                        desc
default_license:                           none
compression_quality:                       90
grants_full_photo_access:                  1
delete_imported:                           0
mod_frame_enabled:                         1
mod_frame_refresh:                         30
thumb_2x:                                  1
small_2x:                                  1
medium_2x:                                 1
landing_page_enable:                       0
force_migration_in_production:             0
search_public:                             0
hide_version_number:                       0
SL_enable:                                 0
SL_for_admin:                              0
recent_age:                                1
grants_download:                           0
photos_wraparound:                         1
map_display:                               0
zip64:                                     1
map_display_public:                        0
map_provider:                              Wikimedia
force_32bit_ids:                           0
map_include_subalbums:                     0
update_check_every_days:                   3
has_exiftool:                              0
share_button_visible:                      0
import_via_symlink:                        0
has_ffmpeg:                                0
location_decoding:                         0
location_decoding_timeout:                 30
location_show:                             1
location_show_public:                      0
rss_enable:                                0
rss_recent_days:                           7
rss_max_items:                             100
prefer_available_xmp_metadata:             0
editor_enabled:                            1
lossless_optimization:                     0
swipe_tolerance_x:                         150
swipe_tolerance_y:                         250
local_takestamp_video_formats:             .avi|.mov
log_max_num_line:                          1000
unlock_password_photos_with_url_param:     0
nsfw_visible:                              1
nsfw_blur:                                 0
nsfw_warning:                              0
nsfw_warning_admin:                        0
nsfw_banner_override:
map_display_direction:                     1
album_subtitle_type:                       oldstyle
upload_processing_limit:                   4
new_photos_notification:                   0
legacy_id_redirection:                     1
zip_deflate_level:                         6
default_album_protection:                  1
album_decoration:                          layers
album_decoration_orientation:              row
allow_username_change:                     1
auto_fix_orientation:                      1
use_job_queues:                            0
random_album_id:                           starred
use_last_modified_date_when_no_exif_date:  0
layout:                                    justified
date_format_photo_thumb:                   M j, Y, g:i:s A e
date_format_photo_overlay:                 M j, Y, g:i:s A e
date_format_sidebar_uploaded:              M j, Y, g:i:s A e
date_format_sidebar_taken_at:              M j, Y, g:i:s A e
date_format_hero_min_max:                  F Y
date_format_hero_created_at:               M j, Y, g:i:s A T
date_format_album_thumb:                   M Y
upload_chunk_size:                         0
nsfw_banner_blur_backdrop:                 0
search_pagination_limit:                   1000
search_minimum_length_required:            4
photo_layout_justified_row_height:         320
photo_layout_masonry_column_width:         300
photo_layout_grid_column_width:            250
photo_layout_square_column_width:          200
photo_layout_gap:                          12
display_thumb_album_overlay:               always
display_thumb_photo_overlay:               hover
default_album_thumb_aspect_ratio:          1/1
use_album_compact_header:                  0
login_button_position:                     left
back_button_enabled:                       0
back_button_text:                          Return to Home
current_job_processing_visible:            1
SA_random_thumbs:                          0
job_processing_queue_visible:              1
login_required:                            0
enable_unsorted:                           1
enable_starred:                            1
enable_recent:                             1
enable_on_this_day:                        1
login_required_root_only:                  1
dark_mode_enabled:                         1
low_quality_image_placeholder:             1
show_keybinding_help_popup:                1
show_keybinding_help_button:               1
disable_se_call_for_actions:               0
enable_se_preview:                         0
slideshow_timeout:                         5
keep_original_untouched:                   1
timeline_photos_enabled:                   1
timeline_photos_public:                    0
timeline_photos_granularity:               day
timeline_photos_order:                     taken_at
timeline_photos_layout:                    square
timeline_photos_pagination_limit:          200
timeline_albums_enabled:                   1
timeline_albums_public:                    0
timeline_albums_granularity:               year
timeline_left_border_enabled:              1
timeline_photo_date_format_year:           Y
timeline_photo_date_format_month:          M Y
timeline_photo_date_format_day:            j M Y
timeline_photo_date_format_hour:           g:i
timeline_album_date_format_year:           Y
timeline_album_date_format_month:          M Y
timeline_album_date_format_day:            j M
hide_nsfw_in_smart_albums:                 1
hide_nsfw_in_frame:                        1
hide_nsfw_in_map:                          1
hide_nsfw_in_search:                       1
search_photos_layout:                      square
hide_nsfw_in_rss:                          1
hide_nsfw_in_timeline:                

### Browser & System [REQUIRED]

Mozilla firefox 115.17.0esr (64-bit)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
Hi, thanks for the report, this seems to be a bug introduced by @aSouchereau placeholder.

Can you go to your settings and switch off : low_quality_image_placeholder to 0 and try again ?

**Reply by nikitakothari06** [reply]
HI,

Disabling low quality image placeholder entry does not resolve the issue in 6.2.0 version

**Comment by prbt2016** [comment]
Hello @ildyria ,

After  disabling Enable low quality image placeholders under All settings i.e :
![image](https://github.com/user-attachments/assets/7950f5a8-40fa-4184-86d2-3c3a3e6304dd)

Images are uploaded properly.

Also one more issue i faced , while under 'Smart albums' .  The '+' icon for upload doesn't work , following is thrown under browser console : 

```
TypeError: a.value is undefined
    r http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/app-BdHWQ6Uc.js:6
    Rs http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    st http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    s http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:18
    Et http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:18
    Qd http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:18
    ip http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:18
    x http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    v http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    F http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    run http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    U http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    W http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    D http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    O http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    E http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    O http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    x http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    v http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    O http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    x http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    v http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    F http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    run http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    U http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    W http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    D http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    O http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    E http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    F http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    run http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    U http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    W http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    D http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    at http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    j http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    E http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    C http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    S http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    v http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    F http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    run http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    runIfDirty http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    Rs http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    hc http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    promise callback*uc http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    Br http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    scheduler http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    trigger http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    Dr http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    notify http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    trigger http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    set value http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    x http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/app-BdHWQ6Uc.js:6
    promise callback*x http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/app-BdHWQ6Uc.js:6
    g http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/app-BdHWQ6Uc.js:6
    promise callback*g http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/app-BdHWQ6Uc.js:6
    E http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/app-BdHWQ6Uc.js:6
    setup http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/app-BdHWQ6Uc.js:6
    Rs http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    Ed http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    fa http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    W http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    D http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    b http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    F http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    run http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    runIfDirty http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    Rs http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    hc http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    promise callback*uc http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    Qs http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:13
    sa http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    scheduler http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:14
    scheduler http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    trigger http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    Dr http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    notify http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    trigger http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    set value http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:9
    Ee http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/vue-router-DLNvcd9X.js:5
    M http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/vue-router-DLNvcd9X.js:5
    promise callback*M http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/vue-router-DLNvcd9X.js:5
    I http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/vue-router-DLNvcd9X.js:5
    g http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/vue-router-DLNvcd9X.js:5
    np http://{{URL}}/{{TO}}/{{LYCHEE}}/public/build/assets/@vue-qHdrsPl8.js:18
@vue-qHdrsPl8.js:13:2006
```

Please check.

Regards.

**Reply by ildyria** [reply]
That plus is not super useful in Smart albums anyway because there are not many action available there, but I'll check.

**Reply by ildyria** [reply]
Fix on its way: https://github.com/LycheeOrg/Lychee/pull/2759

**Comment by nikitakothari06** [comment]
Hi,

I tried installing 6.1.2 version and getting the same issue.

The thumbnails are not loading when uploading images and on creating an album. However when you click on that picture it is there.
![image](https://github.com/user-attachments/assets/ec17a58b-d83d-4896-a839-76b0b1dae825)

I also tried disabling the "Enable low quality images placeholder" setting and the issue persists.

Is the new version designed like this purposely ?

**Reply by ildyria** [reply]
> Is the new version designed like this purposely ?

For sure **no**.

I will try to investigate, it is very weird because I don't have those problems on my version.
Furthermore your issue sounds more like: https://github.com/LycheeOrg/Lychee/discussions/2658

**Comment by nikitakothari06** [comment]
Getting the same issue on 6.2.0 version as well.
